utils.py

```diff
@@ -43,17 +43,17 @@ def post_status(self, status_json):
     def statuses_url(self):
         return self.event['pull_request']['statuses_url']
 
-    def create_status_json(self, required_any, required_all, banned):
+    def create_status_json(self, required_any, required_all, banned, status_text, status_target_url):
         passes_label_requirements = self.validate_labels(required_any, required_all, banned)
         if passes_label_requirements:
             description = "Label requirements satisfied."
         else:
-            description = "Label requirements not satisfied."
+            description = status_text
         response_json = {
             "state": "success" if passes_label_requirements else "failure",
-            "target_url": "",
+            "target_url": status_target_url,
             "description": description,
-            "context": APP_NAME,
+            "context": "compliance/required-labels",
```

Comment:
Seems like we could retain `APP_NAME` here?

Reply:
updated name to `compliance/required-labels` rather than liveramp and kept the "APP_NAME" for the context.